src/s1ap/handlers/s1ap_tau_response_handler.c

```diff
@@ -77,7 +77,7 @@ get_tau_rsp_protoie_value(struct proto_IE *value, struct tauResp_Q_msg *g_tauRes
 #endif
 
 	free(value->data[2].val.nas.elements);
-	free(value->data);
+	//free(value->data);
```

Comment:
I dont see its getting released anywhere else. Can you please tell me the line # at which its getting freed.

Reply:
line 152 and 363. May be this is already updated in the master branch.